cmd/revad/svcs/httpsvcs/ocssvc/users.go

```diff
@@ -48,19 +48,29 @@ func (h *UsersHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	WriteOCSSuccess(w, r, &UsersData{
-		// FIXME query storages? cache a summary?
-		// TODO use list of storages to allow clients to resolve quota status
-		Quota: &QuotaData{
-			Free:       2840756224000,
-			Used:       5059416668,
-			Total:      2845815640668,
-			Relative:   0.18,
-			Definition: "default",
-		},
-		DisplayName: u.DisplayName,
-		Email:       u.Mail,
-	})
+	var head string
+	head, r.URL.Path = httpsvcs.ShiftPath(r.URL.Path)
+	switch head {
+	case "":
+		WriteOCSSuccess(w, r, &UsersData{
+			// FIXME query storages? cache a summary?
+			// TODO use list of storages to allow clients to resolve quota status
+			Quota: &QuotaData{
+				Free:       2840756224000,
+				Used:       5059416668,
+				Total:      2845815640668,
+				Relative:   0.18,
+				Definition: "default",
+			},
+			DisplayName: u.DisplayName,
+			Email:       u.Mail,
+		})
+	case "groups":
+		WriteOCSSuccess(w, r, &GroupsData{})
```

Comment:
for now good enough, this allow login with phoenix again. It seems to die if the element is not present